lxd/db/certificates.go

```diff
@@ -1,10 +1,6 @@
 package db
 
-import (
-	"database/sql"
-
-	_ "github.com/mattn/go-sqlite3"
-)
+import _ "github.com/mattn/go-sqlite3"
```

Comment:
We keep the longer import syntax even for single imports.

Reply:
Ok, I dropped the import entirely, since it was redundant, see 0a3d2445b2ec8ac2646112f04d137b9d3d449db5.

For the record, this kind of changes is not intentional, since I use https://godoc.org/golang.org/x/tools/cmd/goimports which chooses to do that in these situations.